ambari-common/src/main/python/resource_management/libraries/functions/get_user_call_output.py

```diff
@@ -40,10 +41,11 @@ def get_user_call_output(command, user, quiet=False, is_checked_call=True, **cal
   try:
     out_files.append(tempfile.NamedTemporaryFile())
     out_files.append(tempfile.NamedTemporaryFile())
-    
+    uid = pwd.getpwnam(user).pw_uid
     # other user should be able to write to it
     for f in out_files:
-      os.chmod(f.name, 0666)
+      os.chown(f.name,uid,-1)
+      os.chmod(f.name, 0644)
```

Comment:
why not 0600? Any process in system may read output that contains some sensitive data

Reply:
@dlysnichenko ,

Hi the permission is not 600 because this functions is in libraries and we are using it in few places.
one instance is that storm user will be writing the file and ambari-agent user will be reading it.
so its necessary we give 0644 permissions for the /tmp file created.

Reply:
ok, makes sense.